docs/_user_guide/best_practices_r.md

```diff
@@ -31,13 +39,22 @@ getRolls <- function(x) {
 }
 ```
 
-We want to sample from the distribution 100 times to see how many rolls it takes to get a Yahtzee. We can use the `parallel` package, and a distributed apply call `mclapply`:
+We aim to sample from the distribution 100 times to determine how many rolls it takes to achieve a Yahtzee. To accomplish this, we use the `parallel` package, and a distributed apply call `mclapply`:
 
-```r
+```R hl_lines="5" title="R"
+# Load library
 library(parallel)
+
+# Set the number of cores to use
 nc <- 4
+
+# Perform the parallel computation
 res <- mclapply(1:100,getRolls,mc.cores=nc)
+
+# Create a histogram
 hist(unlist(res))
 ```
 
-This uses 4 cores on my local machine to apply this function.
+This uses 4 cores on your local machine to apply this function. If you display or save the results as a histogrm (e.g., `yahtzee_histogram.png`), it will look similar to this:
```

Comment:
I don't think the histogram adds much here, particularly since it doesn't show anything about parallelism.

Also, histogrm is misspelled :)

Reply:
Deleted